conf.pri.msvc

```diff
@@ -0,0 +1,55 @@
+# Adapt these paths
+
+# Point this to the boost include folder
+INCLUDEPATH += $$quote(C:/qBittorrent/boost_1_51_0)
+# Point this to the libtorrent include folder
+INCLUDEPATH += $$quote(C:/qBittorrent/RC_0_16/include)
+# Point this to the zlib include folder
+INCLUDEPATH += $$quote(C:/qBittorrent/Zlib/include)
+# Point this to the openssl include folder
+INCLUDEPATH += $$quote(C:/qBittorrent/openssl/include)
+
+# Point this to the boost lib folder
+LIBS += $$quote(-LC:/qBittorrent/boost_1_51_0/stage/lib)
+# Point this to the libtorrent lib folder
+LIBS += $$quote(-LC:/qBittorrent/RC_0_16/bin/path-according-to-the-build-options-chosen)
+# Point this to the zlib lib folder
+LIBS += $$quote(-LC:/qBittorrent/Zlib/lib)
+# Point this to the openssl lib folder
+LIBS += $$quote(-LC:/qBittorrent/openssl/lib)
+
+# Adapt the lib names/versions accordingly
+CONFIG(debug, debug|release) {
+  LIBS += libtorrentd.lib \
+          libboost_system-vc90-mt-sgd-1_51.lib
```

Comment:
Doesn't boost use automatic linking on MSVC?

Reply:
As I recall, I had problems with it earlier.

Reply:
What is the purpose of `BOOST_SYSTEM_STATIC_LINK=1` above?

Reply:
>What is the purpose of BOOST_SYSTEM_STATIC_LINK=1 above?

I don't know exactly. It's something to do with static linking, so personally I don't use it in my configuration (because of dynamic linking there).

Reply:
>Doesn't boost use automatic linking on MSVC?

I'll try it later.

Reply:
Well, for some reason `BOOST_ALL_NO_LIB` is used to prevent auto-linking. I've moved it to the user config example file so somebody can use auto-linking.
BTW, boost isn't support Visual Studio 2017 for now so it can't auto-link libraries build with this compiler version.